epub33/rs/index.html

```diff
@@ -957,20 +957,18 @@ <h4>Security Considerations</h4>
 
 					<ul>
 						<li>
-							<p>Reading Systems need to behave as if a unique domain were allocated to each Content
-								Document, as browser-based security relies heavily on document URLs and domains.
-								Adopting this approach will isolate documents from each other and from other Internet
-								domains, thereby limiting access to external URLs, cookies, DOM storage, etc.</p>
+							<p>Reading Systems need to behave as if a unique origin were allocated to each <a>EPUB
+									Publication</a>. Adopting this approach will isolate publications from each other,
```

Comment:
I think it is worth putting a reference to the 'formal' origin (ie, to https://url.spec.whatwg.org/#origin) emphasize the Web Security aspect. This can be as simple as making the word an active link or a formal reference to the URL spec (probably the best is to do both).